content/applications/inventory_and_mrp/inventory/product_management/product_replenishment/reordering_rules.rst

```diff
@@ -79,43 +83,187 @@ Reordering Rules`, then click :guilabel:`Create`, and fill out the new line as f
-Orders --> Requests for Quotation`.
+To view and manage :abbr:`RFQs (Requests for Quotation)`, navigate to :menuselection:`Purchase app
+--> Orders --> Requests for Quotation`.
 
 If the :guilabel:`Manufacture` route is selected, then an :abbr:`MO (Manufacturing Order)` is
 generated. To view and manage :abbr:`MOs (Manufacturing Orders)`, navigate to
-:menuselection:`Manufacturing --> Operations --> Manufacturing Orders`.
+:menuselection:`Manufacturing app --> Operations --> Manufacturing Orders`.
+
+When no route is selected, Odoo selects the :guilabel:`Route` specified in the :guilabel:`Inventory`
+tab of the product form.
+
+.. _inventory/product_management/manual-rr:
+
+Manual
+------
+
+Manual reordering rules, configured by setting the reordering rule's :guilabel:`Trigger` field to
+:guilabel:`Manual`, lists a product on the replenishment dashboard when the forecasted quantity
+falls below a specified minimum. Products on this dashboard are called *needs*, because they are
+needed to fulfill upcoming sales orders, but the forecasted quantity is not enough.
 
-Set a preferred route for reordering
-====================================
+The replenishment dashboard, accessible by navigating to :menuselection:`Inventory app -->
+Operations --> Replenishment`, considers sales order deadlines, forecasted stock levels, and vendor
+lead times. It displays needs **only** when it is time to reorder items.
+
+.. note::
+   If the one-day window for ordering products is too short, skip to the :ref:`visibility days
+   <inventory/product_management/visibility-days>` section to make the need appear on the
+   replenishment dashboard a specified number of days in advance.
+
+When a product appears on the replenishment dashboard, clicking the :guilabel:`Order Once` button
+generates the purchase or manufacturing order with the specified amounts :guilabel:`To Order`.
+
+.. image:: reordering_rules/manual.png
+   :align: center
+   :alt: Click the Order Once button on the replenishment dashboard to replenish stock.
+
+.. _inventory/product_management/visibility-days:
+
+Visibility days
+===============
+
+.. note::
+   Ensure :doc:`lead times <../../shipping_receiving/advanced_operations_shipping/scheduled_dates>`
+   are understood before proceeding with this section.
+
+When :ref:`manual reordering rules <inventory/product_management/manual-rr>` are assigned to a
+product, *visibility days* make the product appear on the replenishment dashboard
+(:menuselection:`Inventory app --> Operations --> Replenishment`) a certain number of days in
+advance.
+
+.. example::
+   A product has a manual reordering rule set to trigger when the stock level falls below four
+   units. The current on-hand quantity is ten units.
+
+   The current date is February twentieth, and the *delivery date* on a sales order (in the
+   :guilabel:`Other Info` tab) is March third — twelve days from the current date.
```

Comment:
we can use numbers here; it's more common to write out numbers if they're below 10.
```suggestion
   The current date is February 20th, and the *delivery date* on a sales order (in the
   :guilabel:`Other Info` tab) is March 3rd — twelve days from the current date.
```